### PR TITLE
Slugify on create only

### PR DESCRIPTION
**Pull request by bloveless**
Only slugify a field on create unless always_slugify config is set to true.

### CODE REVIEW

**Comment by bloveless** [comment]
Fixes https://github.com/pyrocms/pyrocms/issues/3928

File: src/SlugFieldType.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ class SlugFieldType extends FieldType
      */
     protected $config = [
         'type'      => '_',
-        'lowercase' => true,
+        'lowercase' => true
```
Comment (RyanThompson) [inline src/SlugFieldType.php]: Can you undo this or update your PHP config? Need to keep end commas everywhere.

File: resources/views/input.twig [hunk 2]
```diff
@@ -17,6 +17,7 @@
             data-type="{{ field_type.config.type }}"
             placeholder="{{ field_type.placeholder }}"
             data-slugify="{{ field_type.config.slugify }}"
+            data-always-slugify="{{ field_type.config.always_slugify }}"
```
Comment (RyanThompson) [inline resources/views/input.twig]: Let's change this to `data-always_slugify` to reflect the configuration key.

File: resources/js/input.js [hunk 3]
```diff
@@ -1,15 +1,18 @@
 $(document).ready(function () {
 
     // Watch for changes if applicable.
+
     $('input[data-provides="anomaly.field_type.slug"]').each(function () {
+        if (!$(this).val() || $(this).data('always-slugify')) {
```
Comment (RyanThompson) [inline resources/js/input.js]: `always_slugify`

**Comment by bloveless** [comment]
That was fast!

**Review by RyanThompson — COMMENTED** [review]
YUUUUS!

**Comment by RyanThompson** [comment]
I've been waiting and refreshing my screen for a few minutes now.

**Pull request merged**